log/context.go

```diff
@@ -62,15 +63,26 @@ func WithLogger(ctx context.Context, logger *logrus.Entry) context.Context {
 }
 
 // GetLogger retrieves the current logger from the context. If no logger is
-// available, the default logger is returned.
+// available, the default logger is returned. If the context has a span
+// associated with it, add correlating information to the returned logger.
 func GetLogger(ctx context.Context) *logrus.Entry {
```

Comment:
Is it going to be confusing hiding the tracing functionality in the `log` pkg and `GetLogger` function?